src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid $light-grey-color;
+  background: $white;
+
+  transition: transform $effect-duration;
+
+  &:hover {
+    @include largeAndChangeColorOnHover;
```

Comment:
Looks like on hover card should change only `transform`, you don't need to change `color` for card on hover